spec/controllers/hubstats/teams_controller_spec.rb

```diff
@@ -33,7 +34,8 @@ module Hubstats
         expect(assigns(:team)).to eq(team)
         expect(pull1.team_id).to eq(team.id)
         expect(pull2.team_id).to eq(team.id)
-        expect(assigns(:team).users).to contain_exactly(user1, user2)
+        expect(assigns(:team).users).to contain_exactly(user1, user2, user2)
```

Comment:
No, this is all as it should be. My goal here is to intentional add `user2` to the team twice. Then the team should have that user on it twice. However, when counting, there should only be 2 users because I added the `distinct`.